site/book/01-getting-started/02-quickstart.md

```diff
@@ -86,14 +99,39 @@ You can preview the changes that will be made to the cluster:
 $ kpt live preview
 ```
 
-Finally, apply the resources to the cluster:
+Apply the resources to the cluster:
 
 ```shell
 $ kpt live apply --reconcile-timeout=15m
 ```
 
 This waits for the resources to be reconciled on the cluster by monitoring their status.
 
+## Fetch updates
+
+On Day n, there might be updates to the remote `nginx` package which you want to rebase your local package against.
```

Comment:
There's some internal jargon which is not appropariate to end-users e.g. "Day n". Simplify the language and correct the grammar.

Reply:
Made is `Subsequently`